.github/workflows/windows.yml

```diff
@@ -63,7 +63,8 @@ jobs:
           conan profile update settings.compiler.runtime=MT${{ matrix.configuration == 'Debug' && 'd' || '' }} default
           # Do not quote the URL. An empty string will be accepted (with
           # a non-fatal warning), but a missing argument will not.
-          conan remote add ripple ${{ env.CONAN_URL }} --insert 0
+          conan remote add ripple ${{ env.CONAN_URL }} --insert 0 || \
+            conan remote list
```

Comment:
What's going on here?

Reply:
The hosted runner seems to be keeping some state between runs, which cause errors when the commands are run again. This was a workaround. Here's an example from my own repo: https://github.com/ximinez/rippled/actions/runs/5490473061/jobs/10005957758#step:8:14

Reply:
Is it because the runner does not create a fresh environment for each job? I think we should consider this a blocker.

Reply:
That's what it looks like.